token-metadata/program/src/processor/uses/utilize.rs

```diff
@@ -30,27 +30,30 @@ pub fn process_utilize(
     number_of_uses: u64,
 ) -> ProgramResult {
     let account_info_iter = &mut accounts.iter().peekable();
+
     let metadata_info = next_account_info(account_info_iter)?;
     let token_account_info = next_account_info(account_info_iter)?;
     let mint_info = next_account_info(account_info_iter)?;
     let user_info = next_account_info(account_info_iter)?;
     let owner_info = next_account_info(account_info_iter)?;
     let token_program_account_info = next_account_info(account_info_iter)?;
     let _ata_program_account_info = next_account_info(account_info_iter)?;
-    let _system_account_info = next_account_info(account_info_iter)?;
+    let _system_program_account_info = next_account_info(account_info_iter)?;
+
```

Comment:
Is the `sysvars` account missing here?

Reply:
It is passed in but not read yet. Technically, these two accounts don't need to be read either because they're not used; I'm not sure of the legacy reason for `_ata_program` to be here.